build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'com.android.library'
+
+dependencies {
+    compile 'com.android.support:support-v4:21.0.3'
```

Comment:
Is this necessary? I thought the support libraries were only needed for backporting features to older Android versions, and I can't think of anything offhand in this project that would need it.